values.yaml

```diff
@@ -351,6 +351,20 @@ svc_sema:
 #            average: 10
 #            burst: 20
 
+# ----------------------------------------------------------------------
+# PROJECT SERVICE CONFIGURATION
+# ----------------------------------------------------------------------
+svc_project:
+  replicas: 4
```

Comment:
can we have the default be 1, and change this on prod to be 4

Reply:
Chen is creating a new PR now. We made all the replica counts as 1 in there. I will update the value-incore-prod.yaml now